doc/getting-started/apl-nuc.rst

```diff
@@ -36,13 +36,13 @@ on your platform. You'll need a network connection for your platform to
 complete this setup.
 
 .. note::
-   ACRN requires Clear Linux version 22140 or newer. The instructions below
-   have been validated with version 22140 and need some adjustment to work
+   ACRN v.01 or latest requires Clear Linux version 23690 or newer. The instructions below
```

Comment:
ACRN release version is ``v0.1`` not ``v.01``

I'd recommend changing this note to be:

    ACRN v0.1 (and the current master branch) requires Clear Linux
    version 23690 or newer.  If you use a newer version of Clear Linux,
    you'll need to adjust the instructions below to reference the version
    number of Clear Linux you are using.